next_review.py

```diff
@@ -13,7 +13,10 @@
 from __future__ import print_function
 
 import argparse
-import ConfigParser
+try:
+    import ConfigParser as configparser
+except ImportError:
+    import configparser
```

Comment:
I didn't want to add another dependency for such little value. The performance difference of values() vs. itervalues() isn't that compelling for a developer tool. The simple try-except is also easy enough to maintain.